src/actions.cc

```diff
@@ -985,10 +987,10 @@ int _action_ranged(Attack* attack, int anim)
 }
 
 // 0x411D68
-int _is_next_to(Object* a1, Object* a2)
+int _is_next_to(Object* obj1, Object* obj2)
 {
-    if (objectGetDistanceBetween(a1, a2) > 1) {
-        if (a2 == gDude) {
+    if (objectGetDistanceBetween(obj1, obj2) > 1) {
+        if (obj2 == gDude) {
```

Comment:
Note: this surprised me a bit, since obj2 is usually the target object when this function is called.